deb-package/pkg/lib/systemd/system/unikernel.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Unikernel Service
+After=network.target
+Before=dvoting.service
```

Comment:
Since `dvoting.service` requires this service, I assume we don't need to have this here.